lib/daru/io/importers/excel.rb

```diff
@@ -45,22 +55,26 @@ class Excel < Base
         #   #    3        4    Franz      nil    Paris      nil
         #   #    4        5   George      5.5     Tome    a,b,c
         #   #    5        6  Fernand      nil      nil      nil
-        def initialize(path, worksheet_id: 0)
+        def initialize(path, sheet: 0, headers: true)
           optional_gem 'spreadsheet', '~> 1.1.1'
```

Comment:
Is it possible to have the `optional_gem` method run at the time of the gem require? Or how about "trying" to require all gems at gem require first and then throw warnings only when the specific method is called?

For example, the `spreadsheet` gem will be required when the user calls `require 'daru/io'` or `require 'daro/io/importers/excel'`. This will not throw any warnings. However, when the user calls `from_excel` you can throw the error. You check whether a library is present or not with this: https://stackoverflow.com/questions/8424769/how-do-you-check-if-a-library-ruby-gem-has-been-loaded

@zverok  thoughts?